webapp/views.py

```diff
@@ -674,6 +674,71 @@ def tutorials_index():
     return tutorials_index
 
 
+def build_engage_index(engage_docs):
```

Comment:
Is there a reason this view can't be part of the discourse module itself?

Reply:
so in a discussion we had about how to structure the module, we decided that what is essentially project stuff would be handled by the project and the module will only handle the data. This is also how it's done on tutorials if you look at above that function currently